test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
Though  I'm still not sure with the behavior but I'm good to go with it now.

`clubbing = False/True` is surely a good thing to have.
`clubbing` is a general term and looks good AFAIK user understands the behavior.